source/triggers/groups/triggerGroup.js

```diff
@@ -13,41 +13,41 @@
 // limitations under the License.
 
 this.initialize = function() {
-    this.triggers = [];
+    this.triggers$ = [];
     this.canFire$ = [];
-    this.checkFrequency$ = 0.08 + ( Math.random() * 0.02 );
-    // this.future( this.checkFrequency$ ).checkTriggers$();
+    this.checkFrequency$ = 0.04 + ( Math.random() * 0.02 );
+    this.future( this.checkFrequency$ ).checkTriggers$();
```

Comment:
This line doesn't need to be uncommented (it can be removed entirely). This call will cause it to only run once on start up. At best it does nothing, at worst it adds a little bit of processing time during load. We start checking triggers consistently when the application state changes to `playing`.